backends/k8s/cert-manager/yaml/cloudflare-apikey-secret.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: cloudflare-api-token-secret
+  name: cloudflare-api-key
   namespace: cert-manager
 type: Opaque
 stringData:
-  api-token: $CLOUDFLARE_API_TOKEN
+  api-key: $CLOUDFLARE_API_KEY
```

Comment:
can you also update `.env.example` and scripts (install.sh, uninstall.sh and investigate-cert.sh)?